docs/command/atlas-kubernetes-operator-install.txt

```diff
@@ -84,7 +88,7 @@ Options
      - string
      - false
      - Namespace where to install the operator.
-   * - --watchNamespace
+   * - --watchNamespaces
```

Comment:
Wouldn't this be a breaking change at this point? Or maybe, we do we support both plural and singular.